lambdas/service/lambda-policy.json.template.py

```diff
@@ -33,6 +33,18 @@
             ],
             "Resource": f"arn:aws:es:{aws.region_name}:{aws.account}:domain/{config.es_domain}"
         },
+        {
+            "Effect": "Allow",
+            "Action": [
+                "sqs:GetQueueAttributes",
+                "sqs:GetQueueUrl",
+                "sqs:ReceiveMessage"
```

Comment:
I don't think this is necessary.

Reply:
So I made some research last night and this is what I found.
[events-sqs-permissions](https://docs.aws.amazon.com/lambda/latest/dg/with-sqs.html#events-sqs-permissions) if you check at the bottom under `Execution Role Permissions` [sqs:ReceiveMessage](https://docs.aws.amazon.com/AWSSimpleQueueService/latest/APIReference/API_ReceiveMessage.html) might be the only one we need. Since we only collect some attributes from the queues. I might have misunderstood the information. @hannes-ucsc